foia_hub/templates/includes/head.html

```diff
@@ -5,6 +5,27 @@
 <link rel="stylesheet" href="{{ static("css/main.css") }}" />
 <link rel="stylesheet" href="{{ static("font-awesome-4.2.0/css/font-awesome.min.css") }}" />
 
+<link rel="apple-touch-icon" sizes="57x57" href="/static/images/favicon/apple-touch-icon-57x57.png">
+<link rel="apple-touch-icon" sizes="60x60" href="/static/images/favicon/apple-touch-icon-60x60.png">
+<link rel="apple-touch-icon" sizes="72x72" href="/static/images/favicon/apple-touch-icon-72x72.png">
+<link rel="apple-touch-icon" sizes="76x76" href="/static/images/favicon/apple-touch-icon-76x76.png">
+<link rel="apple-touch-icon" sizes="114x114" href="/static/images/favicon/apple-touch-icon-114x114.png">
+<link rel="apple-touch-icon" sizes="120x120" href="/static/images/favicon/apple-touch-icon-120x120.png">
+<link rel="apple-touch-icon" sizes="144x144" href="/static/images/favicon/apple-touch-icon-144x144.png">
+<link rel="apple-touch-icon" sizes="152x152" href="/static/images/favicon/apple-touch-icon-152x152.png">
+<link rel="apple-touch-icon" sizes="180x180" href="/static/images/favicon/apple-touch-icon-180x180.png">
+<link rel="icon" type="image/png" href="/static/images/favicon/favicon-32x32.png" sizes="32x32">
+<link rel="icon" type="image/png" href="/static/images/favicon/favicon-194x194.png" sizes="194x194">
+<link rel="icon" type="image/png" href="/static/images/favicon/favicon-96x96.png" sizes="96x96">
+<link rel="icon" type="image/png" href="/static/images/favicon/android-chrome-192x192.png" sizes="192x192">
+<link rel="icon" type="image/png" href="/static/images/favicon/favicon-16x16.png" sizes="16x16">
+<link rel="manifest" href="/static/images/favicon/manifest.json">
+<link rel="shortcut icon" href="/static/images/favicon/favicon.ico">
+<meta name="msapplication-TileColor" content="#ffc40d">
+<meta name="msapplication-TileImage" content="/static/images/favicon/mstile-144x144.png">
+<meta name="msapplication-config" content="/static/images/favicon/browserconfig.xml">
+<meta name="theme-color" content="#ffffff">
+
 <script src="{{ static("js/jquery-2.1.1.min.js") }}"></script>
```

Comment:
so it'll be:
`{{ static("images/favicon/apple-touch-icon-120x120.png") }}`
correct?